go/oasis-test-runner/oasis/fixture.go

```diff
@@ -306,6 +329,10 @@ func (f *ByzantineFixture) Create(net *Network) (*Byzantine, error) {
 	}
 
 	return net.NewByzantine(&ByzantineCfg{
+		NodeCfg: NodeCfg{
+			DisableDefaultLogWatcherHandlerFactories: !f.EnableDefaultLogWatcherHandlerFactories,
```

Comment:
Is the reason for inverting it so that the default is "disabled"? Could we make it consistent in both structures?

Reply:
My reasoning was to enable sensible defaults for different node types.

For example, Byzantine nodes should not have network's default log watcher handler factories enabled because they are byzantine and don't (always) report (their) byzantine behavior in their logs.
[Here](https://buildkite.com/oasislabs/oasis-core-ci/builds/847#f37f7791-1cfe-438c-b897-9479a58dd77c) is an example where honest nodes reported merge discrepancy but `byzantine-0` node didn't.

Reply:
Ok.